eng/Versions.props

```diff
@@ -22,6 +22,18 @@
     <!-- Dependency from https://github.com/dotnet/wpf -->
     <MicrosoftDotNetWpfProjectTemplatesPackageVersion>3.1.0-preview1.19476.4</MicrosoftDotNetWpfProjectTemplatesPackageVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- Dependency from https://github.com/dotnet/templating -->
+    <MicrosoftDotNetCommonItemTemplatesPackageVersion>3.1.0-preview1.19476.7</MicrosoftDotNetCommonItemTemplatesPackageVersion>
+  </PropertyGroup>
+  <PropertyGroup>
+    <!-- Dependency from https://github.com/dotnet/test-templates -->
+    <MicrosoftDotNetTestProjectTemplates31PackageVersion>1.0.2-beta4.19465.1</MicrosoftDotNetTestProjectTemplates31PackageVersion>
+  </PropertyGroup>
+  <PropertyGroup>
+    <!-- NUnit3.DotNetNew.Template versions do not 'flow in' -->
+    <NUnit3TemplatesPackageVersion>1.6.3</NUnit3TemplatesPackageVersion>
```

Comment:
Let's still name this after the package convention (no dots + PackageVersion).

Reply:
Done